### ISSUE TITLE
import visu3d throws @epy.backports.cached_property  AttributeError: module 'etils.epy' has no attribute 'backports'

### ISSUE THREAD

**Issue by relh** [google-research/visu3d#154]
Hi! I'm not sure why but I see this after trying to import visu3d. I'll post here in case I solve it but curious what's wrong. 

python            
Python 3.11.6 | packaged by conda-forge | (main, Oct  3 2023, 10:40:35) [GCC 12.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import visu3d
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/relh/miniforge3/lib/python3.11/site-packages/visu3d/__init__.py", line 37, in <module>
    from visu3d.dc_arrays.camera import Camera
  File "/home/relh/miniforge3/lib/python3.11/site-packages/visu3d/dc_arrays/camera.py", line 29, in <module>
    from visu3d.dc_arrays import camera_spec
  File "/home/relh/miniforge3/lib/python3.11/site-packages/visu3d/dc_arrays/camera_spec.py", line 406, in <module>
    class Spec360(CameraSpec):
  File "/home/relh/miniforge3/lib/python3.11/site-packages/visu3d/dc_arrays/camera_spec.py", line 446, in Spec360
    @epy.backports.cached_property
     ^^^^^^^^^^^^^
AttributeError: module 'etils.epy' has no attribute 'backports'
>>>

**Comment by relh** [comment]
It looks like the most recent commit fixed this by replacing with functools but the pypi verison doesn't have this so I'm just going to build from source:

https://github.com/google-research/visu3d/commit/3cdb8fb40caec9c22afa073fc487805afe3f63c8

**Comment by relh** [comment]
```
(base)  relh@zephyr  ~/Code/visu3d   main  pip install .                                      
WARNING: Skipping /home/relh/miniforge3/lib/python3.11/site-packages/tokenizers-0.15.2.dist-info due to invalid metadata entry 'name'
Processing /home/relh/Code/visu3d
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Requirement already satisfied: dataclass_array in /home/relh/miniforge3/lib/python3.11/site-packages (from visu3d==1.5.3) (1.5.2)
Requirement already satisfied: einops in /home/relh/miniforge3/lib/python3.11/site-packages (from visu3d==1.5.3) (0.8.0)
Requirement already satisfied: etils[edc,enp,epath,epy,etree] in /home/relh/miniforge3/lib/python3.11/site-packages (from visu3d==1.5.3) (1.9.2)
Requirement already satisfied: numpy in /home/relh/miniforge3/lib/python3.11/site-packages (from visu3d==1.5.3) (2.0.0)
Requirement already satisfied: typing_extensions in /home/relh/miniforge3/lib/python3.11/site-packages (from visu3d==1.5.3) (4.12.2)
Requirement already satisfied: lark in /home/relh/miniforge3/lib/python3.11/site-packages (from dataclass_array->visu3d==1.5.3) (1.1.9)
Requirement already satisfied: fsspec in /home/relh/miniforge3/lib/python3.11/site-packages (from etils[edc,enp,epath,epy,etree]->visu3d==1.5.3) (2024.6.1)
Requirement already satisfied: importlib_resources in /home/relh/miniforge3/lib/python3.11/site-packages (from etils[edc,enp,epath,epy,etree]->visu3d==1.5.3) (6.4.0)
Requirement already satisfied: zipp in /home/relh/miniforge3/lib/python3.11/site-packages (from etils[edc,enp,epath,epy,etree]->visu3d==1.5.3) (3.19.2)
Requirement already satisfied: absl-py in /home/relh/miniforge3/lib/python3.11/site-packages (from etils[edc,enp,epath,epy,etree]->visu3d==1.5.3) (2.1.0)
Requirement already satisfied: tqdm in /home/relh/miniforge3/lib/python3.11/site-packages (from etils[edc,enp,epath,epy,etree]->visu3d==1.5.3) (4.66.4)
Building wheels for collected packages: visu3d
  Building wheel for visu3d (pyproject.toml) ... done
  Created wheel for visu3d: filename=visu3d-1.5.3-py3-none-any.whl size=69092 sha256=f3bdd5c0f76dc7fa0ae77b271973a2280b1ebb3adf391f6b13518464448bd7d7
  Stored in directory: /tmp/pip-ephem-wheel-cache-lbtjxp61/wheels/6d/31/7a/9739b6f9a154293f451c7b7451272f8b91e193ba06b95bf368
Successfully built visu3d
WARNING: Skipping /home/relh/miniforge3/lib/python3.11/site-packages/tokenizers-0.15.2.dist-info due to invalid metadata entry 'name'
Installing collected packages: visu3d
  Attempting uninstall: visu3d
    WARNING: Skipping /home/relh/miniforge3/lib/python3.11/site-packages/tokenizers-0.15.2.dist-info due to invalid metadata entry 'name'
    Found existing installation: visu3d 1.5.3
    Uninstalling visu3d-1.5.3:
      Successfully uninstalled visu3d-1.5.3
WARNING: Skipping /home/relh/miniforge3/lib/python3.11/site-packages/tokenizers-0.15.2.dist-info due to invalid metadata entry 'name'
Successfully installed visu3d-1.5.3
WARNING: Skipping /home/relh/miniforge3/lib/python3.11/site-packages/tokenizers-0.15.2.dist-info due to invalid metadata entry 'name'
WARNING: Skipping /home/relh/miniforge3/lib/python3.11/site-packages/tokenizers-0.15.2.dist-info due to invalid metadata entry 'name'
(base)  relh@zephyr  ~/Code/visu3d   main  python       
Python 3.11.6 | packaged by conda-forge | (main, Oct  3 2023, 10:40:35) [GCC 12.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> imoprt visu3d
  File "<stdin>", line 1
    imoprt visu3d
           ^^^^^^
SyntaxError: invalid syntax
>>> import visu3d
>>> 
```

For anyone else, building from source worked

**Comment by ignacio-rocco** [comment]
This issue should be fixed with 1.5.4, for 1.5.3, you can use etils==1.7.0. @Conchylicultor
